ci/lint/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# See test/lint/README.md for usage.
+#
+# This container basically has to live in this directory in order to pull in the CI
+# install scripts. If it lived in the root directory, it would have to pull in the
+# entire repo as docker context during build; if it lived elsewhere, it wouldn't be
```

Comment:
For reference, you can avoid passing the full context by using buildkit or podman. See

* https://docs.docker.com/build/buildkit/#overview
* https://docs.podman.io/en/latest/markdown/podman-build.1.html#name